eventcollectors/src/main/kotlin/events/boudicca/eventcollector/collectors/FamilienkarteEventCollector.kt

```diff
@@ -0,0 +1,67 @@
+package events.boudicca.eventcollector.collectors
+
+import events.boudicca.SemanticKeys
+import events.boudicca.api.eventcollector.Event
+import events.boudicca.api.eventcollector.Fetcher
+import events.boudicca.api.eventcollector.TwoStepEventCollector
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Element
+import java.time.LocalDate
+import java.time.LocalTime
+import java.time.OffsetDateTime
+import java.time.ZoneId
+import java.time.format.DateTimeFormatter
+
+class FamilienkarteCollector : TwoStepEventCollector<String>("familienkarte") {
+
+    private val fetcher = Fetcher()
+
+
+    override fun getAllUnparsedEvents(): List<String> {
+        val document = Jsoup.parse(fetcher.fetchUrl("https://www.familienkarte.at/de/freizeit/veranstaltungen/veranstaltungskalender.html?events_cat_key=8"))
+        return document.select("div.detailButton a")
+                .map { it.attr("href") }
+    }
+
+    override fun parseEvent(event: String): Event {
+        val eventSite = Jsoup.parse(fetcher.fetchUrl(event))
+
+        val name = eventSite.select("div.eventDetailWrapper h1").text()
+        val (startDate, endDate) = parseDates(eventSite)
+
+        val data = mutableMapOf<String, String>()
+        data[SemanticKeys.URL] = event
+        if (endDate != null) {
+            data[SemanticKeys.ENDDATE] = endDate.format(DateTimeFormatter.ISO_DATE)
+        }
+        data[SemanticKeys.TYPE] = "other"
```

Comment:
type other does not really make sense, if you don't have any information on the type of event just don't add it :)